### PR TITLE
Add title_placeholder param to PageHeader pattern

### PR DESCRIPTION
**Pull request by mleray**
### Description

This is needed for some pattern layouts such as [High Level Topic](https://jira.greenpeace.org/browse/PLANET-6737) or [Campaign](https://jira.greenpeace.org/browse/PLANET-6528). This commit actually also updates the High Level topic one, which you can use for testing these changes!

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [phobos](https://www-dev.greenpeace.org/test-phobos) | [admin](https://www-dev.greenpeace.org/test-phobos/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-phobos/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-phobos) | [composer-local.json](https://github.com/greenpeace/planet4-test-phobos/blob/main/composer-local.json)

:watch: 2022.08.09 13:43:04

File: classes/patterns/class-pageheader.php [hunk 1]
```diff
@@ -38,10 +38,11 @@ public static function get_name(): string {
...
 	 */
 	public static function get_config( $params = [] ): array {
-		$classname  = 'is-pattern-p4-page-header';
-		$media_link = esc_url( get_template_directory_uri() ) . '/images/placeholders/placeholder-546x415.jpg';
-		$pos_class  = 'left' === static::$media_position ? ''
+		$classname         = 'is-pattern-p4-page-header';
+		$media_link        = esc_url( get_template_directory_uri() ) . '/images/placeholders/placeholder-546x415.jpg';
+		$pos_class         = 'left' === static::$media_position ? ''
 			: 'has-media-on-the-' . static::$media_position;
+		$title_placeholder = $params['title_placeholder'] ?? '';
```
Comment (lithrel) [inline classes/patterns/class-pageheader.php:45]: To be more precise, it's the title, or title content, not the placeholder

Comment (mleray) [reply classes/patterns/class-pageheader.php:45]: True, true, but it is used as a placeholder since it's meant to be changed 🤔

Comment (lithrel) [reply classes/patterns/class-pageheader.php:45]: :thinking:  I see your point, mine was just to not confuse it with the `placeholder` attribute, but that works too :+1:

**Review by lithrel — APPROVED** [review]
:+1:

**Pull request merged**